python/pyspark/cloudpickle.py

```diff
@@ -280,7 +279,7 @@ def extract_code_globals(co):
         # see if nested function have any global refs
         if co.co_consts:
             for const in co.co_consts:
-                if type(const) is types.CodeType:
+                if isinstance(const, types.CodeType):
```

Comment:
Hm, i think this is third-party code. I'm reluctant to change it because it's just a straight copy of the file from a particular release. Can we exclude it from pep8?

You're welcome of course to file a bug against cloudpickle.